src/main/java/flink/graphs/Graph.java

```diff
@@ -699,13 +734,14 @@ public void remove() {
 
 	/**
 	 * Creates a graph from a dataset of vertices and a dataset of edges
-	 * @param vertices
-	 * @param edges
-	 * @return
+	 * @param vertices a flink Dataset of vertices.
```

Comment:
I would prefer simply "DataSet" instead of "flink Dataset" (what other kind could it be?)